docs/userGuide/markBindSyntaxOverview.md

```diff
@@ -110,15 +113,21 @@ MarkBind has several core Vue.js components built on the popular [BootStrap](htt
 ```
 {% endraw %}
 
-{{ icon_arrow_down }}
+</div>
+
+%%OUTPUT:%%
+<div class="indented">
+
+<box border-left-color={{ markbind_blue }} background-color="white">
```

Comment:
May I know if this part of the `codeAndOutput.md` boilerplate appears blue on your end? The default boilerplate seems to appear grey on my end:

![Capture](https://user-images.githubusercontent.com/25716529/219940228-9c42e564-d4d9-4924-8376-7fd37e3daa92.PNG)

Or is there a reason why we are changing it to blue specifically?

Reply:
Apologies, fixed in [479dd7d](https://github.com/MarkBind/markbind/pull/2175/commits/479dd7d548217f8a59a6355ac4e430090d93c9dd). I was working on a PR which changes it to blue